packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -149,6 +150,92 @@ export class ProductCard extends PureComponent {
         );
     }
 
+    renderBundlePriceBadge(bounds) {
+        const {
+            product: { type_id, price_view }
+        } = this.props;
+
+        if (type_id !== BUNDLE) {
+            return null;
+        }
+
+        // eslint-disable-next-line
+        let bundleBadgeLabel;
+
+        if (bounds === 'bottom') {
+            bundleBadgeLabel = 'From';
+
+            if (price_view !== 'PRICE_RANGE') {
+                bundleBadgeLabel = 'As low as';
+            }
+        }
+
+        if (bounds === 'top') {
+            bundleBadgeLabel = 'To';
+        }
+
+        return (
+            <p
+              mix={ {
+                  block: 'ProductCard',
+                  elem: 'PriceBadge'
+              } }
+            >
+                { __(bundleBadgeLabel) }
+            </p>
+        );
+    }
+
+    renderOldMaxBundlePrice() {
+        const {
+            product: { price_range }
+        } = this.props;
+
+        const oldMaxBundlePrice = `${ price_range.maximum_price.regular_price.value }`;
+        const finalMaxBundlePrice = `${ price_range.maximum_price.final_price.value }`;
```

Comment:
Please use destructuring and don't forget to specify defaults.
https://docs.scandipwa.com/structure/code-style/javascript-code-style#use-destructuring

Reply:
Also you don't shouldn't use the template literals if you have a single value.